CHANGELOG.md

```diff
@@ -12,6 +12,30 @@
 #### Removed
 #### Documentation
 
+## [v0.8.0](https://github.com/mesg-foundation/core/releases/tag/v0.8.0)
+
+#### Breaking Changes
+
+- ([#690](https://github.com/mesg-foundation/core/pull/690)) Support service deployments from tarball urls. To continue to select a branch with a git repository URL, make sure that the URL start with `git://`.
```

Comment:
We still support git for now so this is not a breaking change, we actually added the support of tarball.
```bash
➜  core git:(dev) ✗ mesg-core service deploy https://github.com/mesg-foundation/service-ethereum\#parity-dev
✔ Service downloaded with success
✔ Service context received with success
✔ Image built with success
✔ Service deployed with hash: 62d225b3ad2f627d1cca2f07f08c90c948fe4e4b
To start it, run the command:
        mesg-core service start 62d225b3ad2f627d1cca2f07f08c90c948fe4e4b
➜  core git:(dev) ✗ mesg-core service deploy https://github.com/mesg-foundation/service-ethereum
✔ Service downloaded with success
✔ Service context received with success
✔ Image built with success
✔ Service deployed with hash: 3aa05c4545e76411bbccd48794409ef881ba8f14
To start it, run the command:
        mesg-core service start 3aa05c4545e76411bbccd48794409ef881ba8f14
```